openvm/src/lib.rs

```diff
@@ -1649,13 +1653,15 @@ mod tests {
                     log_up: 3952,
                 },
             }),
-        });
-
-        // Assert that total columns don't exceed the initial limit set
-        let total_columns = (powdr_metrics_sum + NON_POWDR_EXPECTED_SUM).widths.total();
-        assert!(
-            total_columns <= MAX_TOTAL_COLUMNS,
-            "Total columns exceeded the limit: {total_columns} > {MAX_TOTAL_COLUMNS}"
         );
+
+        // TODO
```

Comment:
Is this important? Can I remove it? It doesn't really work like that any more because the expectations are strings now.